invenio_assets/assets/.eslintrc.js

```diff
@@ -0,0 +1,29 @@
+/*
+ * This file is part of Invenio.
+ * Copyright (C) 2017-2018 CERN.
+ *
+ * Invenio is free software; you can redistribute it and/or modify it
+ * under the terms of the MIT License; see LICENSE file for more details.
+ */
+
+module.exports = {
+  root: true,
+  parser: 'babel-eslint',
+  parserOptions: {
+    sourceType: 'module'
+  },
+  env: {
+    browser: true,
+  },
+  // https://github.com/feross/standard/blob/master/RULES.md#javascript-standard-style
+  extends: 'react-app',
```

Comment:
does this have any impact on the current angular apps?

Reply:
I think we need to test this with a new cookiecutter instance, because in my tests in app ILS I have removed search-ui.